### PR TITLE
#21077 adding ability to the local events to be cluster wide

### PR DESCRIPTION
**Pull request by jdotcms**
Adding the ability to local events to be cluster wide at server level

### CODE REVIEW

File: dotCMS/src/main/java/com/dotcms/rest/api/v1/system/logger/ChangeLoggerLevelEvent.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
+package com.dotcms.rest.api.v1.system.logger;
+
+public class ChangeLoggerLevelEvent {
```
Comment (freddyucv) [inline dotCMS/src/main/java/com/dotcms/rest/api/v1/system/logger/ChangeLoggerLevelEvent.java:3]: javadoc

File: dotCMS/src/main/java/com/dotcms/rest/api/v1/system/logger/LoggerResource.java [hunk 2]
```diff
@@ -112,22 +120,40 @@ public Response changeLoggerLevel(@Context final HttpServletRequest request,
...
+                }
+            }
+
+            // queue the change logger level to trigger the local event on the rest of the cluster
+            APILocator.getSystemEventsAPI().
+                    queue(SystemEventType.LOCAL_SYSTEM_EVENT,
+                            new Payload(
+                                    new ChangeLoggerLevelEvent(
+                                            changeLoggerForm.getName(), changeLoggerForm.getLevel()
+                                    )));
```
Comment (freddyucv) [inline dotCMS/src/main/java/com/dotcms/rest/api/v1/system/logger/LoggerResource.java:152]: should the Resource trigger the event?

**Review by freddyucv — APPROVED** [review]

File: dotCMS/src/main/java/com/dotcms/rest/api/v1/system/logger/LoggerResource.java (hunk 2 above)
Comment (jdotcms) [reply dotCMS/src/main/java/com/dotcms/rest/api/v1/system/logger/LoggerResource.java:152]: agree, but there is not API for Logger.

File: dotCMS/src/main/java/com/dotcms/api/system/event/SystemEventType.java [hunk 3]
```diff
@@ -26,6 +26,8 @@
  */
 public enum SystemEventType {
 
+	LOCAL_SYSTEM_EVENT,
```
Comment (wezell) [inline dotCMS/src/main/java/com/dotcms/api/system/event/SystemEventType.java:29]: This should be `CLUSTER_WIDE_EVENT`   I would also put javadocs on each of the enum properties

**Review by wezell — DISMISSED** [review]

**Review by erickgonzalez — APPROVED** [review]

**Comment by erickgonzalez** [comment]
We need to merge master to this branch, so the tests run. We need to check the tests results before merging

**Pull request merged**